autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -452,13 +452,16 @@ def leaderboard(self, X=None, y=None, silent=False):
                 print(leaderboard)
         return leaderboard
 
-    # TODO: Should this return a dictionary or pandas Series? Current it returns a Series
     # TODO: enable_fit_continuation must be set to True to be able to pass X and y as None in this function, otherwise it will error.
     # Warning: This can take a very, very long time to compute if the data is large and the model is complex.
     # TODO: Warning: This is not memory safe, large datasets may result in OOM errors during feature importance calculation.
+    # TODO: Rewrite description before adding to Predictor
     # A value of 0.01 means that the objective metric error would be expected to increase by 0.01 if the feature were removed.
     # Negative values mean the feature is likely harmful.
-    def get_feature_importance(self, model, X=None, y=None) -> Series:
+    # model: model (str) to get feature importances for, if None will choose best model.
+    # features: list of feature names that feature importances are calculated for and returned, specify None to get all feature importances.
+    # raw: Whether to compute feature importance on raw original features or on the features used by the particular model.
+    def get_feature_importance(self, model=None, X=None, y=None, features: list = None, raw=True) -> Series:
```

Comment:
For exposing to user, we might consider naming `raw` more informatively, like `raw_features` or `original_features`, 
or having two functions `get_feature_importance` (for raw features only) `get_input_importance` (for all model inputs),
or even just solely exposing `raw=True` functionality, since non-raw seems primarily for pruning.

Reply:
Agreed